data/data/rhcos.json

```diff
@@ -111,11 +111,11 @@
             "size": 647639040
         },
         "qemu": {
-            "path": "rhcos-43.80.20191002.1-qemu.x86_64.qcow2",
-            "sha256": "29e867427e84b64970b8e0408b135b13f6e315b82882be5601e19529d4463773",
-            "size": 700315704,
-            "uncompressed-sha256": "b4cd810376dca356f987e2e7f8609f84d63720d5ad19d3101f84e82d1918c85e",
-            "uncompressed-size": 1889337344
+            "path": "fedora-coreos-30.20191014.0-qemu.x86_64.qcow2.xz",
```

Comment:
This is going to break baremetal, we rely on the QEMU image for the bootstrap node - it seems odd to have mixed RHCOS/FCOS content here, shouldn't it be one or the other for everything?

Reply:
Also the next time someone runs the hack script to update the RHCOS build this will all get overwritten.

Reply:
Baremetal IPI is currently not supported on OKD-on-FCOS - Ironic bits need to be replaced with UBI8 + RDO, hack script updated etc. I'll make another PR once it gets there

Reply:
Right I get that it won't work with OKD-on-FCOS, but this is breaking baremetal IPI entirely. Is all of openshift/install master moving to FCOS and too bad for any platforms that don't work with it yet?

Reply:
My concern here is baremetal IPI has been broken for weeks, and we're very close to recovering from it and having working CI again. Now, with this change, our Metal3 CI can't be run against anything in master, so we really have no idea if some future commits break us until it ends up in a release. :-(

Reply:
Sorry for the noise, it was just pointed out to me this isn't going to the master branch

Reply:
> it seems odd to have mixed RHCOS/FCOS content here, shouldn't it be one or the other for everything?

@vrutkovs I think this very important question from @stbenjam wasn't answered. If the answer is that we're just moving everything to FCOS, I think we should rename the file and (more importantly) commit log should reflect the reason for the change.

Reply:
This PR is only for the `fcos` branch, it won't affect main development on `master`.